batch/src/main/java/com/hartwig/batch/input/FlatInputParser.java

```diff
@@ -0,0 +1,39 @@
+package com.hartwig.batch.input;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FlatInputParser implements InputParser {
+    private String inputFile;
```

Comment:
both implementations take inputfile and billedproject as fields, perhaps best to move these into arguments to parse.

Reply:
Good catch. Done.